src/configmanager.cpp

```diff
@@ -334,6 +364,19 @@ float ConfigManager::getExperienceStage(uint32_t level) const
 	return std::get<2>(*it);
 }
 
+float ConfigManager::getLoyaltyBonus(uint16_t points) const
+{
+	auto it = std::find_if(loyaltyBonuses.begin(), loyaltyBonuses.end(), [points](LoyaltyBonuses::value_type bonus) {
```

Comment:
`auto it = std::find_if(loyaltyBonuses.begin(), loyaltyBonuses.end(), [points](const auto& bonus) {`?